src/main/java/racingcar/controller/WebController.java

```diff
@@ -0,0 +1,34 @@
+package racingcar.controller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import racingcar.dto.RequestDto;
+import racingcar.dto.ResponseDto;
+import racingcar.model.Cars;
+import racingcar.service.GameService;
+import racingcar.util.NameFormatConverter;
+
+import java.util.List;
+
+@Controller
+public class WebController {
+
+    private final GameService gameService;
+
+    @Autowired
+    public WebController(GameService gameService) {
+        this.gameService = gameService;
+    }
+
+    @PostMapping("/plays")
+    public ResponseEntity<ResponseDto> playGame(@RequestBody RequestDto requestDto) {
+        List<String> carNames = NameFormatConverter.splitNameByDelimiter(requestDto.getNames());
+        Cars cars = new Cars(carNames);
+
+        gameService.executeRacingGame(cars, requestDto.getCount());
+        return ResponseEntity.ok().body(new ResponseDto(cars.getWinners(), cars.getCars()));
```

Comment:
```java
	/**
	 * A shortcut for creating a {@code ResponseEntity} with the given body
	 * and the status set to {@linkplain HttpStatus#OK OK}.
	 * @param body the body of the response entity (possibly empty)
	 * @return the created {@code ResponseEntity}
	 * @since 4.1
	 */
	public static <T> ResponseEntity<T> ok(@Nullable T body) {
		return ok().body(body);
	}
```
요걸 사용할 수 있을 것 같군요

Reply:
다양한 메소드를 익히고 사용법을 익혀보겠습니다..
위의 메소드가 더 효율적인 것 같아 사용하였습니다!